### PR TITLE
Support multi environments

### PR DESCRIPTION
**Pull request by moshe-kabala**
- Remove binary file
- Refactor auth pkg - support multiple envs
- Integrate the cli commands, and add auth select to switch between envs
- Integrate the installation login
- Integrate ctx creation

### CODE REVIEW

**Comment by yotamazriel** [comment]
Please provide more details on the functionality you added in this pr

**Comment by moshe-kabala** [comment]
Sure, all the comments are on `auth`
`Login` - Create environment (the env name created from the URL by default)
`Logout` - Remove the current environment, unless the user sends the environment name
`Select` - Switch between environment

File: pkg/auth/config.go [hunk 1]
```diff
@@ -1,31 +1,134 @@
...
+	ENVIRONMENTS_CONFIG_PATH = "envs"
+	AUTH_CONFIG_PATH         = "auth"
 
-func GetApiUrl() string {
-	return viper.GetString("auth.api_url")
+	CURRENT_ENV = "current_env"
+)
+
+var (
+	// just for backward compatibility, will use just current_env and envs in the future
```
Comment (YuvalFeldman) [inline pkg/auth/config.go:21]: Maybe you should open a task for this so it doesnt become a forever todo?

**Review by YuvalFeldman — APPROVED** [review]
LGTM, very cool feature

**Pull request merged**